src/editor/serialize.ts

```diff
@@ -38,10 +42,43 @@ export function mdSerialize(model: EditorModel) {
 }
 
 export function htmlSerializeIfNeeded(model: EditorModel, {forceHTML = false} = {}) {
-    const md = mdSerialize(model);
+    let md = mdSerialize(model);
+
+    if (SettingsStore.getValue("feature_latex_maths")) {
+        const displayPattern = (SdkConfig.get()['latex_maths_delims'] || {})['display_pattern'] ||
+            "\\$\\$\\$(([^$]|\\\\\\$)*)\\$\\$\\$";
+        const inlinePattern = (SdkConfig.get()['latex_maths_delims'] || {})['inline_pattern'] ||
+            "\\$\\$(([^$]|\\\\\\$)*)\\$\\$";
```

Comment:
Seeing as both fluffy chat (https://gitlab.com/famedly/famedlysdk/-/merge_requests/501) and the Element-Android PR (https://github.com/vector-im/element-android/pull/2133) adhere to "standard" LaTeX conventions rather than Zulip's, it might be better to default to that instead. I've had to remind people to use double `$$` very frequently for inline maths so far because they are so used to `$x + y$`, as opposed to Zulip conventions.

Reply:
done in 839bae2. Since this is now a "labs" feature that users can switch off and on, i think it's reasonable to set it to single $'s by default